content/webapp/pages/search/works.tsx

```diff
@@ -57,7 +57,7 @@ const SortPaginationWrapper = styled.div`
   flex-wrap: wrap;
 `;
 
-export const CatalogueSearchPage: NextPageWithLayout<Props> = ({
+const CatalogueSearchPage: NextPageWithLayout<Props> = ({
```

Comment:
[double export](https://github.com/wellcomecollection/wellcomecollection.org/blob/9a569d2a3e8ae7c2f45cdf8f36360be70c526c42/content/webapp/pages/search/works.tsx#L320), so removed it here.